profiling/simple-kernel-timer/kp_shared.h

```diff
@@ -39,6 +42,15 @@ inline bool compareKernelPerformanceInfo(KernelPerformanceInfo* left,
   return left->getTime() > right->getTime();
 };
 
+inline int find_index(const std::vector<KernelPerformanceInfo*>& kernels,
+                      const char* kernelName) {
+  const auto it =
+      std::find_if(kernels.begin(), kernels.end(), [&](const auto& kernel) {
+        return strcmp(kernel->getName(), kernelName) == 0;
+      });
+  return it == kernels.cend() ? -1 : std::distance(kernels.begin(), it);
```

Comment:
I see little benefit from changing the original implementation
```suggestion
  for (unsigned int i = 0; i < kernels.size(); i++) {
    KernelPerformanceInfo* nextKernel = kernels[i];

    if (strcmp(nextKernel->getName(), kernelName) == 0) {
      return i;
    }
  }

  return -1;
```

Reply:
I meant to apply the suggestion and merge but somehow I don't see the option to do so in the GH interface.